docker-compose build with stac-browser >= 2.0.0-rc.3

Some changes have been introduced in [stac-browser v2.0.0-rc.3](https://github.com/radiantearth/stac-browser/releases/tag/v2.0.0-rc.3) which make docker build fail and/or be deprecated:
- `yargs` usage
- Deprecate env variables (e.g. `CATALOG_URL=https://landsat-stac.s3.amazonaws.com/catalog.json npm start -- --open`)
```sh
$ docker-compose build

[...]
Step 6/10 : RUN CATALOG_URL=http://localhost:5000 npm run build
 ---> Running in eceeca8ae9a9

> @radiantearth/stac-browser@2.0.0 build /stac-browser
> vue-cli-service build

 ERROR  Error loading /stac-browser/vue.config.js:
 ERROR  Error: yargs supports a minimum Node.js version of 12. Read our version support policy: https://github.com/yargs/yargs#supported-nodejs-versions
Error: yargs supports a minimum Node.js version of 12. Read our version support policy: https://github.com/yargs/yargs#supported-nodejs-versions
    at Object.<anonymous> (/stac-browser/node_modules/yargs/build/index.cjs:1:55294)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/stac-browser/node_modules/yargs/index.cjs:5:30)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @radiantearth/stac-browser@2.0.0 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @radiantearth/stac-browser@2.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-08T15_25_16_531Z-debug.log

```

--- a/docker/stac-browser.dockerfile
+++ b/docker/stac-browser.dockerfile
@@ -1,5 +1,5 @@
 # build stage
-FROM node:lts-jessie-slim as build-stage
+FROM node:lts-buster-slim as build-stage
 
 # install git
 RUN apt-get update \
@@ -15,7 +15,7 @@ WORKDIR /stac-browser
 RUN npm install
 
 # start application
-RUN CATALOG_URL=http://localhost:5000 npm run build
+RUN npm run build -- --CATALOG_URL=http://localhost:5000
 
 # production stage, self describing
 FROM nginx:stable-alpine as production-stage

